src/firebase/firebase.config.ts

```
@@ -0,0 +1,14 @@
+export default () => {
+  const firebaseConfigString = process.env.FIREBASE_CONFIG;
+
+  if (!firebaseConfigString) {
+    throw new Error('FIREBASE_CONFIG environment variable not found.');
+  }
+
+  try {
+    const config = JSON.parse(firebaseConfigString);
+    return config;
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)